test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -817,8 +817,19 @@ context('action sidebar', function() {
       .should('have.length', 1);
 
     cy
-      .get('body')
-      .type('{esc}');
+      .get('.picklist')
+      .find('.js-picklist-item')
+      .first()
+      .click();
+
+    cy
+      .get('.five9-panel')
+      .should('have.class', 'is-open');
+
+    cy
+      .get('.five9-panel__header')
+      .find('[data-status-region]')
+      .click();
```

Comment:
Just clicks on a picklist item and verifies the panel is opened via the radio request. Not sure how else we want to test this at the moment.